ginivision/src/doc/source/changelog.rst

```diff
@@ -2,6 +2,16 @@
 Changelog
 =========
 
+2.1.0 (2017-01-30)
+==================
+
+- Removed the 4:3 aspect ratio camera resolution requirement. Only a minimum of 8MP is required.
```

Comment:
+- Removed the 4:3 aspect ratio requirement for photos. The default camera aspect ratio will be used from now on. An 8MP minimum resolution is still required.